tests/suite/resources_utils.py

```diff
@@ -352,11 +352,12 @@ def get_service_node_ports(v1: CoreV1Api, name, namespace) -> (int, int, int, in
     :return: (plain_port, ssl_port, api_port, exporter_port)
     """
     resp = v1.read_namespaced_service(name, namespace)
-    assert len(resp.spec.ports) == 5, "An unexpected amount of ports in a service. Check the configuration"
+    assert len(resp.spec.ports) == 6, "An unexpected amount of ports in a service. Check the configuration"
```

Comment:
although it was a pre-existing one, but can we change assert to if-else as assert can cause an exit if false

Reply:
fixed